x/ibc/module.go

```diff
@@ -58,12 +58,21 @@ func (AppModuleBasic) DefaultGenesis(_ codec.JSONMarshaler) json.RawMessage {
 
 // ValidateGenesis performs genesis state validation for the ibc module.
 func (AppModuleBasic) ValidateGenesis(cdc codec.JSONMarshaler, bz json.RawMessage) error {
-	var gs GenesisState
-	if err := cdc.UnmarshalJSON(bz, &gs); err != nil {
-		return fmt.Errorf("failed to unmarshal %s genesis state: %w", ModuleName, err)
-	}
 
-	return gs.Validate()
+	// TODO: UNDO this when DefaultGenesis() is implemented
```

Comment:
https://github.com/cosmos/cosmos-sdk/issues/5948

Reply:
As @anilCSE has noted here, there is an issue with IBC `ValidateGenesis` that was breaking cli tests. I suggest we merge this PR with `ValidateGenesis` commented out like this so that cli tests can proceed. Then the IBC team fixes this issue, they can uncomment these lines.

Reply:
ACK

Reply:
Is there a currently open issue / PR for tracking this and making sure this doesn't get forgotten? #5948 seems to be closed, and I cant find any open related issues.

Reply:
Threre's none. May be we can reopen this issue: https://github.com/cosmos/cosmos-sdk/issues/6082

Reply:
> there is an issue with IBC ValidateGenesis that was breaking cli tests.

which one?

Reply:
>DefaultGenesis is not implemented and it just returns nil

DefaultGenesis is defined in `x/ibc/genesis.go`

Reply:
https://github.com/cosmos/cosmos-sdk/blob/master/x/ibc/module.go#L55-L77